apps/src/lib/levelbuilder/ai-tutor/AITutorTester.tsx

```diff
@@ -63,12 +69,46 @@ const AITutorTester: React.FC<AITutorTesterProps> = ({allowed}) => {
     for (let i = 0; i < data.length; i++) {
       if (selectedEndpoint === 'llm-guard') {
         getLLMGuardToxicity(data[i]);
+      } else if (genAIEndpointIds.includes(selectedEndpoint)) {
+        getGenAIResponses(data[i]);
       } else {
-        askAI(data[i]);
+        askAITutor(data[i]);
       }
     }
   };
 
+  const getGenAIResponses = async (row: AIInteraction) => {
+    const systemPrompt = row.systemPrompt ? row.systemPrompt : '';
+    const chatMessage = {
+      chatMessageText: row.studentInput,
+      role: Role.USER,
+      status: 'ok',
+      timestamp: new Date().getTime(),
+    };
+    const temperature = row.temperature ? row.temperature : 0.8;
```

Comment:
Nit: Could we make this a constant and add a comment that it's the default temperature used in levelbuilder?